peer_mentoring/tests/test_groups.py

```diff
@@ -59,6 +69,33 @@ def group(db, avatar, category, user):
     return group
 
 
+@pytest.fixture
+def join_request(db, user, user2, group):
+    return GroupJoinRequest.objects.create(
+        sender=user, receiver=group.moderator, group=group
+    )
+
+
+# Fixture to make post instance
+@pytest.fixture
+def post(db, user, group):
+    return Post.objects.create(
+        author=user, title="Test Post Title", content="Test Post Content", group=group
+    )
+
+
+@pytest.fixture
+def comment(db, user, post):
```

Comment:
ditto

Reply:
Yes, it wasn't finding it otherwise, but corrected with the conftest.py